std_msgs/Header header
s4_msgs/TrackedObject[] objects